openshift-client/src/main/java/io/fabric8/openshift/client/OpenshiftSubjectAccessOperations.java

```diff
@@ -13,18 +13,10 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+package io.fabric8.openshift.client;
 
-package io.fabric8.openshift.client.dsl;
-
-import io.fabric8.kubernetes.api.model.Doneable;
-import io.fabric8.kubernetes.client.dsl.Createable;
-import io.fabric8.openshift.api.model.SubjectAccessReview;
-import io.fabric8.openshift.api.model.SubjectAccessReviewFluentImpl;
 import io.fabric8.openshift.api.model.SubjectAccessReviewResponse;
 
-public abstract class CreateableSubjectAccessReview extends
-  SubjectAccessReviewFluentImpl<CreateableSubjectAccessReview>
-  implements
-  Doneable<SubjectAccessReviewResponse>,
-  Createable<SubjectAccessReview, SubjectAccessReviewResponse, CreateableSubjectAccessReview> {
+public interface OpenshiftSubjectAccessOperations<T> {
```

Comment:
Different approach to https://github.com/fabric8io/kubernetes-client/pull/2400#discussion_r474025692

Again, why not just ditch the interface and use Creatable instead in the implementing class?